app/src/main/java/com/codingblocks/cbonlineapp/dashboard/mycourses/DashboardMyCoursesFragment.kt

```diff
@@ -45,7 +46,8 @@ class DashboardMyCoursesFragment : BaseCBFragment(), AnkoLogger {
                 startActivity(
                     intentFor<MyCourseActivity>(
                         RUN_ATTEMPT_ID to runAttemptId,
-                        COURSE_NAME to name
+                        COURSE_NAME to name,
+                        COURSE_ID to id
```

Comment:
There are n ways to open a activity and passing course id shouldn't be part of that and if you do make sure it doesn't break elsewhere